.travis.yml

```diff
@@ -9,21 +9,45 @@ git:
 matrix:
   include:
     - os: linux
-      env: GCC_VERSION=7
+      env:
+        - GCC_VERSION=7
+        - cmake_option="-DREPORTS_ENABLE_MPI=ON -DCMAKE_C_COMPILER=mpicc -DCMAKE_CXX_COMPILER=mpicxx"
```

Comment:
cool, thanks for doing this; I haven't read it carefully, but it seems like the right direction on first glance :+1: